app/templates/components/project-geometries/modes/draw.hbs

```diff
@@ -3,8 +3,8 @@
     {{fa-icon 'draw-polygon' size='2x' fixedWidth=true}}
     {{#ember-tooltip side='right' tooltipClassName="ember-tooltip draw-tooltip" popperContainer='body'}}
       <h5 class="small-margin-bottom">Polygon</h5>
-      <p class="small-margin-bottom">Add a new shape. Complete it by clicking the final point a 2nd time.</p>
-      <p class="no-margin">Edit an existing shape by selecting it and dragging/deleting its points. Add a new point by clicking the midpoint of a line.</p>
+      <p class="small-margin-bottom">Add a new shape. Complete it by clicking the final corner a 2nd time.</p>
+      <p class="no-margin">Edit an existing shape by selecting it and dragging/deleting its corners. Add a new point by clicking the midpoint of a line.</p>
```

Comment:
I'd already changed these from "vertex" to "point." I think point might be better because sometimes these vertices make very slight angles that don't look like "corners". And it's nice if adding "a new point" uses the same noun.